CpEpEngine.cpp

```diff
@@ -2595,28 +2591,27 @@ STDMETHODIMP CpEpEngine::SignatureVerifies(BSTR text, BSTR signature, VARIANT_BO
     return status;
 }
 
-static void cache_passphrase(const string& passphrase)
-{
-    passphrase_cache.add(passphrase);
-}
-
 STDMETHODIMP CpEpEngine::ManagePassphrase(LPSAFEARRAY accounts_with_old_passphrases, BSTR new_passphrase, LPSAFEARRAY* error_accounts) {
     string passphrase = utf8_string(new_passphrase);
     stringpair_list_t* account_passphrases = stringpair_list(accounts_with_old_passphrases);
     stringlist_t* _error_accounts = NULL;
 
     const PEP_STATUS status = manage_passphrase(session(), account_passphrases, passphrase.c_str(), &_error_accounts);
 
-    free_stringpair_list(account_passphrases);
-
     if (status != PEP_STATUS_OK && _error_accounts) {
         *error_accounts = string_array(_error_accounts);
     }
     else if (status == PEP_STATUS_OK) {
-        // cache the new passphrase
-        cache_passphrase(passphrase);
```

Comment:
I see we don't cache passphrase without email anymore, I will update that and verify behavior.

Reply:
At least not in this part of the API. Is still supported though by the passphrase cache, and can still get triggered. E.g. by "losing a key sync".